services/jwt-auth/src/main/java/com/gentics/mesh/auth/util/KeycloakUtils.java

```diff
@@ -148,5 +202,4 @@ private static OkHttpClient httpClient() {
 		OkHttpClient client = builder.build();
 		return client;
 	}
```

Comment:
The ultimate question - why we have something Keycloak-related in Vanilla Mesh anyway, while it should be agnostic of any identity provider?

Reply:
Most probably because most of the KC-integration tests are baked into Mesh UT suite...